plugins/dev-scripts/config/webpack.config.js

```diff
@@ -120,6 +120,7 @@ module.exports = (env) => {
       // Run the linter.
       !env.skipLint && new ESLintPlugin({
         cache: true,
+        cacheLocation: path.join('node_modules/.cache/eslint/'),
```

Comment:
Isn't the more standard way to do this to pass each directory as a different parameter? E.g:

```suggestion
        cacheLocation: path.join('node_modules', '.cache', 'eslint'),
```

Reply:
You can do that too, but you don't need to. It will automatically normalize the path segments and switch them to the appropriate operating-system-specific separator even if they're in one string. So I didn't see a reason to split them up; that makes more sense when parts of the path are variables as opposed to string literals.